kadalu_operator/main.py

```diff
@@ -1062,6 +1176,26 @@ def main():
     core_v1_client = client.CoreV1Api()
     k8s_client = client.ApiClient()
 
+    user_create_cmd = ["kadalu","user", "create", "admin", "--password=kadalu"]
```

Comment:
Add `TODO` to remove the hardcoded password. Also, add `TODO` when a change is to be done with later PRs.